.gitignore

```diff
@@ -15,3 +15,4 @@
 /log/*
 !/log/.keep
 /tmp
+.DS_Store
```

Comment:
This is a pretty common thing to do, but after a while local ignore files get pretty cluttered if lots of different people with lots of different operating systems and editors work on it. As a rule, we think local ignore files should only contain files that pertain to the repository itself.

That's not to say we should version control `.DS_Store`! You can put that in your system ignore file, though, which we think make more sense because really it's about your system and not the repository.